### ISSUE TITLE
Import Folders

### ISSUE THREAD

**Issue by doonoo** [sviatoslav-lebediev/anki-quizlet-importer-extended#167]
Would be nice to import folders. For example this https://quizlet.com/mirandawallace/folders/comptia-a-1101

**Comment by sviatoslav-lebediev** [comment]
Hi, thank you for the feature request. I may add it when I have time. If you can create a PR, feel free to do this, and I will review + merge.

**Comment by doonoo** [comment]
Yeah, I'm not sure how to even do a pull request, I know just enough to
report on GitHub haha. Thank you for your work though sir, it's surely
appreciated.

**Comment by sviatoslav-lebediev** [comment]
It has been implemented
